source/connect/client.txt

```diff
@@ -0,0 +1,110 @@
+.. _php-client:
+
+=======================
+Create a MongoDB Client
+=======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoDB\\Client** object, which creates the connection to the MongoDB deployment
+  and lets you perform operations on it.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoDB\Client`` object
+to connect to MongoDB.
```

Comment:
Q: does this guide show how to create a connection string or does it just teach about connection strings?

Reply:
it describes the components of the connection string

Reply:
Do you think we need to be clearer about this point? I hoped that if we showed users all of the pieces of the connection string, they could put one together...?

Reply:
I dont think many users are creating a connection string from its constituent pieces. Perhaps this sentence could express something more like:
```suggestion
This guide describes the components of a connection string and shows how to
 use a ``MongoDB\Client`` object to connect to a MongoDB deployment.
```